### PR TITLE
Add PDF to Keynote.

### PR DESCRIPTION
**Pull request by LumingYin**
1. [x] Project URL:
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Screenshot included 
5. [x] Avoid iOS or open-source in description as it is assumed
6. [x] Use this commit title format if applicable: Add app-name by @github-username
7. [x] Use approved format for your entry

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -17730,6 +17730,20 @@
         "https://is2-ssl.mzstatic.com/image/thumb/Purple128/v4/05/68/18/0568186a-f883-1172-de2c-e3dbfb9adaae/mzl.oqskbgaz.png/690x0w.jpg",
         "https://is2-ssl.mzstatic.com/image/thumb/Purple118/v4/4a/5c/09/4a5c097b-df61-fe5c-7d81-97f757d7ad74/mzl.yzivdnlw.png/690x0w.jpg"
       ]
+    },
+    {
+      "title": "PDF to Keynote",
+      "category-ids": ["media", "education"],
+      "description": "Import PDF presentations as vector images into Keynote for iOS.",
+      "source": "https://github.com/LumingYin/PDFToKeynote-iOS",
+      "homepage": "https://github.com/LumingYin/PDFToKeynote-iOS/blob/master/README.md",
```
Comment (dkhamsing) [inline contents.json]: `homepage` is not needed if same as`source`, please remove

File: contents.json [hunk 2]
```diff
@@ -17730,6 +17730,20 @@
         "https://is2-ssl.mzstatic.com/image/thumb/Purple128/v4/05/68/18/0568186a-f883-1172-de2c-e3dbfb9adaae/mzl.oqskbgaz.png/690x0w.jpg",
         "https://is2-ssl.mzstatic.com/image/thumb/Purple118/v4/4a/5c/09/4a5c097b-df61-fe5c-7d81-97f757d7ad74/mzl.yzivdnlw.png/690x0w.jpg"
       ]
+    },
+    {
+      "title": "PDF to Keynote",
+      "category-ids": ["media", "education"],
```
Comment (dkhamsing) [inline contents.json]: Seems one category is good enough, please remove `education`

**Comment by LumingYin** [comment]
> Pls make the suggested changes, thank you!

Updated the pull request per suggested changes.

**Comment by dkhamsing** [comment]
Thx!

**Pull request merged**